public/app/features/api-keys/state/reducers.ts

```diff
@@ -31,10 +38,14 @@ const apiKeysSlice = createSlice({
     isFetching: (state): ApiKeysState => {
       return { ...state, hasFetched: false };
     },
+    migrationResult: (state, action): ApiKeysState => {
```

Comment:
It's better to use something like setMigrationResult or migrationResultUpdated
```suggestion
    setMigrationResult: (state, action): ApiKeysState => {
```